rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
+  
+#### 58 Notice of meetings
 
-5. If the number of nominations exceeds the number of vacancies to be filled, a ballot  must be held. 
+1. If a special committee meeting is convened, the Secretary must give Committee Members notice by way of email which must include the general nature of the business to be conducted.
 
-6. The ballot for the election of officers and ordinary members of the committee must be  conducted at the general meeting in such manner as the committee may direct. 
+2. The only business that may be conducted at the meeting is the business for which the meeting is convened.
 
-## 25. Vacancies
+#### 59 Urgent meetings
 
-The office of an officer of the Association, or of an ordinary member of the committee, becomes vacant if the officer or member—
+1. In cases of urgency, a meeting can be held without notice being given in accordance with rule 58 provided that as much notice as practicable is given to each committee member by the quickest means practicable.
 
-1. ceases to be a member of the Association; or 
+2. Any resolution made at the meeting must be passed by an absolute majority of the Committee.
 
-2. becomes an insolvent under administration within the meaning of the Corporations Act; or 
+3. The only business that may be conducted at an urgent meeting is the business for which the meeting is convened.
 
-3. resigns from office by notice in writing or email given to the Secretary. 
+#### 60 Procedure and order of business
 
-## 26. Meetings of the committee
+1. The procedure to be followed at a meeting of a Committee must be determined from time to time by the Committee.
 
-1. The committee must meet at least 2 times in each year at such place and such times as the committee may determine. 
+2. The order of business may be determined by the members present at the meeting.
 
-2. Special meetings of the committee may be convened by the President or by any 4 members of the committee. 
+#### 61 Quorum
 
-## 27. Notice of committee meetings
+1. No business may be conducted at a Committee meeting unless a quorum is present.
 
-1. Written or email notice of each committee meeting must be given to each member of the committee at least 2 business days before the date of the meeting. 
+2. The quorum for a committee meeting is the presence of a majority of the committee members holding office.
 
-2. Written or email notice must be given to members of the committee of any special meeting specifying the general nature of the business to be conducted and no other business may be conducted at such a meeting. 
+3. If a quorum is not present within 30 minutes after the notified commencement time of a committee meeting—
+  1. in the case of a special meeting—the meeting lapses;
+  2. in any other case—the meeting must be adjourned to a date no later than 14 days after the adjournment and notice of the time, date and place to which the meeting is adjourned must be given in accordance with rule 59.
```

Comment:
This references Rule 59, but the relevant rule appears to be Rule 58 (Notice of meetings). Can someone confirm this is correct and update this reference?